docs/docsite/rst/playbook_guide/playbooks_debugger.rst

```diff
@@ -278,7 +278,7 @@ After you update the module argument, use ``redo`` to run the task again with th
 Update vars command
 -------------------
 
-``task_vars[*key*] = *value*`` updates the ``task_vars``. You could fix the playbook above by viewing, then updating the task variables instead of the module args.
+``task_vars[*key*] = *value*`` updates the ``task_vars``. You could fix the playbook above by viewing, and then updating the task variables instead of the module args.
```

Comment:
```suggestion
``task_vars[*key*] = *value*`` updates the ``task_vars``. You could fix the playbook above by viewing and then updating the task variables instead of the module args.
```